### DISCUSSION TITLE
Need help testing Blazored Toast with bunit

### DISCUSSION THREAD

**Discussion by anitagov** [Blazored/Toast#116] [Q&A] [answered]
@chrissainty - I posted my query in the bunit Discussions forum. Here is the link:
https://github.com/bUnit-dev/bUnit/discussions/412
Great if I can get some help with testing this with C#/bunit.
Thanks.

- Anita

**Comment by chrissainty** [comment]
Hi @anitagov, we're actually working on adding bUnit testing support right now. I'll post an update here when it lands.

**Reply by anitagov** [reply]
This is great news @chrissainty . Do you have a timeline on when this will be available?

**Reply by chrissainty** [reply]
I'm hoping, in the next couple of weeks,

**Comment by anitagov** [comment]
Hi @chrissainty - Is this ready for use? A few things are missing currently making it non-trivial to use right now.
1) Readme.md on how to use the Blazored Toast extension for bunit.
2) A nuget package with Blazored.Toast.TestExtension will definitely help.

Thanks for your kind response.

**Reply by chrissainty** [reply]
Yes, the Blazored Toast TestExtensions package was published last week. There is also a sample project showing it's use in the repo (https://github.com/Blazored/Toast/tree/main/samples/bUnitExample)

**Reply by anitagov** [reply]
I was able to find the Blazored.Toast.TestExtensions nuget package via Visual Studio IDE and could not find it earlier. I am able to use this package and get the tests to work with these test extensions. Thanks @chrissainty and @OliverBack.

**Comment by anitagov** [comment, marked as answer]
Use the Blazored.Toast.TestExtensions nuget package.
